packages/plugin/src/rules/federation-subgraph.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  asFed2SubgraphDocument,
+  buildSchemaFromAST,
+  FederationBlueprint,
+  Subgraph,
+} from '@apollo/federation-internals';
+import { GraphQLESLintRule } from '../types.js';
+
+const RULE_ID = 'federation-subgraph';
```

Comment:
any suggestions to rule name?

Reply:
`apollo-federation-subgraph` would be more precise?